src/main/java/domain/Car.java

```diff
@@ -0,0 +1,28 @@
+package domain;
+
+public class Car {
+
+    private String name;
+    private int position = 0;
+
+    public Car(String name) {
+        this.name = name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void moveCar(int random) {
+        if (random >= 4) {
+            this.position += 1;
+        }
+    }
+
+    public Boolean isWinner(int winningPosition) {
+        if (this.position == winningPosition) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
이 메소드 시그니처가 직관적이지 않은데요
이 메소드가 Car 클래스에 있어야 할까? 하는 생각이 드는 것 같아요

제가 생각하는 객체지향을 사용하는 가장 큰 장점은 특정 클래스 or 메소드만 보고서도 코드의 흐름을 파악하기 쉬운 것이 큰 장점인 것 같은데요

이때 이 Car 클래스의 메소드인 isWinner 라는 것은 약간 애매한 것 같아요
아래 방식 2가지중 하나로 바꿔보면 어떻게 생각하시는지 궁금합니다

1.  저라면 Racing 클래스에서 .filter(car -> car.getPosition()==winningPosition) 와 같은 형태로 진행할 것 같아요


2번은 suggestion 으로 작성해두었습니다

```suggestion
    public boolean hasSamePosition(int position) {
        if (this.position == position) {
            return true;
        }
        return false;
    }
```
코드를 읽는 입장에서 isWinner 로 되어있는 코드를 읽는 편과 단순 getter 를 쓰거나 hasSamePosition 같은 네이밍이면 조금 더 인지에 들어가는 피로도를 줄일 수 있을 것 같아요

저는 메소드 네이밍이나 가독성을 크게 신경쓰는 편은 아니라 의견이 다를 수 있어요
제 의견들은 어떻게 생각하시는지 달아주시면 감사하겠습니다!
마음에 든다면 수정해주셔도 좋고, 아니라면 유지해주셔도 좋아요

Reply:
지금 생각해 보니  .filter(car -> car.getPosition()==winningPosition) 로 처리해도 가독성있게 읽을 수 있는 코드를 불필요하게 isWinning 이라는 메서드를 만든 것 같다고 생각이 드네요

isWinning이라는 메서드의 네이밍이 너무 Racing 클래스의 비즈니스에 특화된 네이밍인것 같고, 게다가 이 메서드가 Car 클래스에 위치하니 더욱 이해하기 힘드셨을 것 같습니다

또한 isWinning 메서드가 Racing 클래스 내에 위치해 있다고 하더라도, isWinning라고 네이밍을 하게 되면 해당 메서드를 재사용하기 어려워 지는 것 같네요

말씀해 주신대로 수정을 하는게 맞는것 같습니다! 감사합니다